lib/ansible/modules/files/iso_extract.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2013, Jeroen Hoekx <jeroen.hoekx@dsquare.be>
+# (c) 2016, Matt Robinson <git@nerdoftheherd.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'core',
+                    'version': '1.0'}
+
+DOCUMENTATION = r'''
+---
+author: Jeroen Hoekx (@jhoekx)
+module: iso_extract
+short_description: Extract files from an ISO image.
+description:
+  - This module mounts an iso image in a temporary directory and extracts
+    files from there to a given destination.
+version_added: "2.3"
+options:
+  image:
+    description:
+    - The ISO image to extract files from
+    required: true
+  dest:
+    description:
+    - The destination directory to extract files to.
+    required: true
+  files:
+    description:
+    - A comma separated list of files to extract from the image.
```

Comment:
Bit of a nitpick & not sure if this is the right time for this, but should this just read 'A list of...' as both comma separated and actual lists in YAML are supported?